cookbook-react/src/recipes/selectors/selector-2/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+import './styles.scss';
+
+interface Option {
+  text: string;
+  disabled: boolean;
+  id: number;
+}
+
+interface Props {
+  options: Option[];
+}
+
+function Selector({
+  // TODO: Remove this options in a real app
+  options = [
+    { id: 1, text: 'Opción 1', disabled: false },
+    { id: 2, text: 'Opción 2', disabled: false },
+    { id: 3, text: 'Opción 3', disabled: false }
+  ]
+}: Props) {
+  const [active, setActive] = useState<null | number>(null);
+  return (
+    <div className="options-container">
+      {options.map((option, idx) => (
+        <button
+          className={cn('btn', { selected: active === idx, 'one-item': options.length === 1 })}
```

Comment:
the rule if the `options.length` is one could be achieved using only one css selector `only-child` https://developer.mozilla.org/en-US/docs/Web/CSS/:only-child